dist/cjs/data/frameworks.json

```diff
@@ -30,6 +30,12 @@
       "outputLocation": "./",
       "configFiles": ["index.html", "Index.html", "default.htm", "default.html", "index.htm"]
     },
+    {
+      "id": "docker-file",
+      "name": "Dockerfile",
+      "outputLocation": "./",
+      "configFiles": ["Dockerfile", "docker-compose.yml", "docker-compose-dev.yml", "docker-compose.images.yml", "docker-stack.yml"]
```

Comment:
don't need this update, can we remove this please? 
the only one needed is here - src/data/frameworks.json

Reply:
Done